crates/polars-plan/src/dsl/function_expr/range/utils.rs

```diff
@@ -7,3 +8,12 @@ pub(super) fn temporal_series_to_i64_scalar(s: &Series) -> i64 {
         .extract::<i64>()
         .unwrap()
 }
+
+pub(super) fn ensure_range_bounds_contain_single_value(
+    start: &Series,
+    end: &Series,
+) -> PolarsResult<()> {
+    polars_ensure!(start.len() == 1, ComputeError: "`start` must contain a single value");
```

Comment:
maybe include `"got {n}"` in the message?

Reply:
Done!